### PR TITLE
Submission of dismo and rgbif

### PR DESCRIPTION
**Pull request by sylvainschmitt**
This is the submission of dismo, dsmextra, MinBAR and rgbif as the authors are not authors of the manuscript in preparation. I also added a new function `template_from_cran` to help build a submission in R. As I am not the author of these packages, I will not detail the features, as I would prefer the reviewers to check the original documentation to see if I am not wrong.

### CODE REVIEW

**Comment by sylvainschmitt** [comment]
I'm looking for reviewers: @EduardoArle , @gepinillab , @hannahlowens , @jamiemkass , @sgvignali, @codelab3 , @adamlilith; @MayaGueguen are you available? Please have a look at the [reviewing guidelines](https://github.com/sylvainschmitt/sdmverse/blob/main/REVIEWING.md)

**Comment by jamiemkass** [comment]
I can review here.

File: R/template_from_cran.R [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  file <- read_yaml(file.path(system.file(package = "sdmverse"),
+                              "extdata", "packages", "example.yaml"))
+  file$name <- cran$name
+  file$title <- cran$title
+  file$version <- cran$version
+  file$author <- cran$author
+  file$maintainer <- cran$maintainer
+  file$cran <- TRUE
+  file$description <- cran$description
+  if(!is.null(path))
```
Comment (jamiemkass) [inline R/template_from_cran.R]: Would be better to rewrite this as an if/else.

**Comment by sylvainschmitt** [comment]
@jamiemkass , as discussed I removed dsmextra and MinBAR. And I followed your suggestion for the template_from_cran function. I'm waiting for your review of the fields to merge the PR.

**Comment by sylvainschmitt** [comment]
@EduardoArle , @gepinillab , @hannahlowens , @sgvignali, @codelab3 , @adamlilith, @MayaGueguen, @danlwarren , @AMBarbosa , @AndreaSanchezTapia , I'm still searching for a second reviewer to help Jamie. As this is not my packages I unfortunately did not described every fields, so this a bit more work as usual. But as  `rgbif` and `dismo` are very known packages I hope it will be easy for some of you.

File: inst/extdata/packages/dismo.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: dismo
+title: Species Distribution Modeling
+version: 1.3-9
+author: Robert J. Hijmans, Steven Phillips, John Leathwick, Jane Elith
+maintainer: Robert J. Hijmans <r.hijmans@gmail.com>
+cran: yes
+repository: ""
+description: Methods for species distribution modeling, that is, predicting the environmental
+  similarity of any site to that of the locations of known occurrences of a species.
+occ_acquisition: no
```
Comment (jamiemkass) [inline inst/extdata/packages/dismo.yaml]: dismo has a `gbif` function that qualifies as occ_acquisition.

File: inst/extdata/packages/dismo.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+occ_cleaning: no
+data_integration: no
+env_collinearity: no
+env_process: no
+bias: no
```
Comment (jamiemkass) [inline inst/extdata/packages/dismo.yaml]: The `ssb` function address sampling bias.

File: inst/extdata/packages/dismo.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+study_region: no
+backg_sample: yes
+data_partitioning: yes
+mod_fit: yes
+mod_tuning: no
```
Comment (jamiemkass) [inline inst/extdata/packages/dismo.yaml]: The `gbm` function does tune BRTs.

File: inst/extdata/packages/dismo.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+mod_ensemble: no
+mod_stack: no
+mod_evaluate: yes
+mod_multispecies: no
+mod_mechanistic: no
+pred_general: yes
+pred_extrapolation: no
```
Comment (jamiemkass) [inline inst/extdata/packages/dismo.yaml]: The `mess` function helps to visualize model extrapolation.

File: inst/extdata/packages/dismo.yaml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+pred_inspect: no
```
Comment (jamiemkass) [inline inst/extdata/packages/dismo.yaml]: The `response` function makes marginal response curves.

File: inst/extdata/packages/rgbif.yaml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+repository: https://github.com/ropensci/rgbif
+description: A programmatic interface to the Web Service methods provided by the Global
+  Biodiversity Information Facility (GBIF; <https://www.gbif.org/developer/summary>).
+  GBIF is a database of species occurrence records from sources all over the globe.
+  rgbif includes functions for searching for taxonomic names, retrieving information
+  on data providers, getting species occurrence records, getting counts of occurrence
+  records, and using the GBIF tile map service to make rasters summarizing huge amounts
+  of data.
+occ_acquisition: yes
+occ_cleaning: no
```
Comment (jamiemkass) [inline inst/extdata/packages/rgbif.yaml]: It actually does this now: http://cran.nexr.com/web/packages/rgbif/vignettes/issues_vignette.html

**Review by jamiemkass — CHANGES_REQUESTED** [review]
Some edits to be made.

**Comment by sylvainschmitt** [comment]
> Some edits to be made.

Thanks! All done.

**Comment by sylvainschmitt** [comment]
> Some edits to be made.

Can you approve the changes and validate a new review for me to merge?

**Comment by jamiemkass** [comment]
Sorry, looks like that wasn't necessary? I'll do this next time.

<http://andersonlab.ccny.cuny.edu>

**Pull request merged**